css/style.css

```diff
@@ -455,5 +457,14 @@ footer {
 
   .footer-filter {
     grid-column-start: 1;
+    padding-top: 20px;
   }
+
+  .footer-toggle-buttons {
+    grid-template-columns: 1fr;
+  }
+}
+
+.unimplemented {
+  display: none;
```

Comment:
how about changing this to `visibility: hidden` instead? That way, the DOM structure isn't broken and probably we won't have to restructure the UI, unless there are strong reasons to change from grid to flex :)